### PR TITLE
Updating Power Plant Costing Documentation

### PR DESCRIPTION
**Pull request by bpaul4**
## Changes proposed in this PR:
- Updates documentation for changes in #978 
- General changes to index files

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/reference_guides/model_libraries/power_generation/costing/index.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Power Plant Costing Libraries
+=============================
+
+The IDAES Process Modeling Framework contains a library of models specifically developed for modeling power generation systems. These models all built off of the core IDAES modeling framework and model libraries.
```
Comment (AlexNoring) [inline docs/reference_guides/model_libraries/power_generation/costing/index.rst:4]:
```suggestion
The IDAES Process Modeling Framework contains a library of models specifically developed for costing power generation systems. These models all build off of the core IDAES modeling framework and model libraries.
```

File: docs/reference_guides/model_libraries/power_generation/costing/power_plant_costing_netl.rst [hunk 2]
```diff
@@ -21,16 +22,21 @@ The first function (get_PP_costing) can be called to include cost correlations f
 6. natural gas air-fired plant (NGCC),
 7. Advanced ultra-supercritical PC (AUSC).
 
-Similarly, `get_sCO2_unit_cost` can be called to include cost correlations for equipment in supercritical CO2 power cycle plants.
+Similarly, `get_sCO2_unit_cost` can be called to include cost correlations for equipment in supercritical CO2 power cycle plants and the method `get_ASU_cost` calls costing for equipment in air separation units. The method `costing_initialization` exists to initialize pp, sCO2 and ASU costing blocks.
+
+The methods `get_fixed_OM_costs` and `get_variable_OM_costs` calculate operating and maintenance costs for fixed (operating labor, maintenance labor, admin/support labor, property taxes and insurance, maintenance materials) and variable (fuel, consumable and waste disposal costs for electricity production indexed by time), respectively. The methods `initialize_fixed_OM_costs` and `initialize_variable_OM_costs` exist to initialize fixed and variable O&M costing blocks. To build all operating and maintanence costs sequentially, users may call the `build_process_costs` method.
```
Comment (AlexNoring) [inline docs/reference_guides/model_libraries/power_generation/costing/power_plant_costing_netl.rst:27]:
```suggestion
The methods `get_fixed_OM_costs` and `get_variable_OM_costs` calculate operating and maintenance costs for fixed (operating labor, maintenance labor, admin/support labor, property taxes and insurance, maintenance materials) and variable (fuel, consumable and waste disposal costs for electricity production indexed by time) costs, respectively. The methods `initialize_fixed_OM_costs` and `initialize_variable_OM_costs` exist to initialize fixed and variable O&M costing blocks. To build all operating and maintanence costs sequentially, users may call the `build_process_costs` method.
```

File: docs/reference_guides/model_libraries/power_generation/costing/power_plant_costing_netl.rst [hunk 3]
```diff
@@ -305,45 +319,158 @@ The last one is for the scaled parameter divided by n_equip.
...
 * scaled_param - The scaled parameter. For the ASU it is the oxygen flowrate in units of tons per day.
+* CE_index_year : Chemical Engineering Cost Index base year, defaults to 2018; calling the registered Pyomo currency units dictionary of plant cost index values will allow conversion between base years within the flowsheet
+
+The bare erected and total plant costs are calculated as shown in the introduction.
+There are currently no estimates for the total plant cost components, so bare erected cost will be the same as total plant cost for now.
+
+Fixed Operating & Maintenance Costs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Fixed O&M costing function adds constraints to calculate correlations associated with labor, maintenance, support, taxes and other fixed costs. The method takes the following arguments:
```
Comment (AlexNoring) [inline docs/reference_guides/model_libraries/power_generation/costing/power_plant_costing_netl.rst:359]:
```suggestion
The Fixed O&M costing function adds constraints to estimate the labor, maintenance, support, taxes and other fixed costs. The method takes the following arguments: 
```

**Comment by bpaul4** [comment]
Sorry reviewers for all of the emails, GitHub wouldn't let me batch all three commits from @AlexNoring .

**Review by AlexNoring — APPROVED** [review]
lgtm once test passes

**Review by andrewlee94 — APPROVED** [review]

**Pull request merged**